docs/guides/end-to-end-testing/testing-strategies/azure-active-directory-authentication.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Azure Active Directory Authentication
+slug: /guides/end-to-end-testing/azure-active-directory-authentication
+---
+
+:::info
+
+## <Icon name="graduation-cap"></Icon> What you'll learn
+
+- Log in to
+  [Azure Active Directory](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-whatis)
+  through the UI with [`cy.origin()`](/api/commands/origin)
+
+:::
+
+:::caution
+
+This guide is designed for testing against a Single Page Application (SPA) that
+uses
+[Azure Active Directory](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-whatis)
```

Comment:
```suggestion
[Azure Active Directory](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-whatis) (AAD)
```
you use this abreviate below without calling out what it stands for

Reply:
I thought I had it in the Microsoft AAD App set up but I reference AAD first before actually defining it 🤦🏻

Reply:
addressed in 7e426fc8a